web-client/iron-svelte-client/src/lib/login/login.svelte

```diff
@@ -6,14 +6,14 @@
     import { toast } from '$lib/messages/message-store';
     import { showLogin } from '$lib/login/login-store';
     import type { DesktopSize } from '../../models/desktop-size';
-
```

Comment:
You should leave the newline before the `import`s

Reply:
updated